dask/array/reshape.py

```diff
@@ -136,10 +136,13 @@ def reshape(x, shape):
     This is a parallelized version of the ``np.reshape`` function with the
     following limitations:
 
-    1.  It assumes that the array is stored in C-order
+    1.  It assumes that the array is stored in `column-major order`_
```

Comment:
Maybe "stored in C-order (sometimes referred to as `column-major order`_)"?

Reply:
I think we decided C-order is row-major order. I like this suggestion, +1 for adding "(sometimes referred to as `row-major order`_)"